real-time/real-time_test.js

```diff
@@ -363,5 +363,39 @@ if (canDev) {
 		}, 500);
 		connection.getList({ "fooBar": true, foo: { bar: 1234 }});
 	});
+
+	test("listSet algebra warning includes any `undefined` values", function() {
+		var algebra = new set.Algebra(set.comparators.id("id"));
+		var items = [{id: 1, name:"d", foo: undefined }, {id: 3, name:"j", foo: {bar: 5678}}];
+		var dataBehavior = function(){
+			return {
+				getListData: function(){
+					// nothing here first time
```

Comment:
Can you change this comment to be more accurate? I know it was copied from other tests that also have confusing comments, but might as well fix it for this one.